packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 44.4.0 - November 1, 2018
+
+### Fixed
+
+#### salesforcedx-vscode-apex-replay-debugger
+
+- We fixed some minor under-the-hood bugs.
```

Comment:
The refactor of replay debugger config is too detailed so opted for a generic fixes message. The rest of this week's changes are doc stuff.